api/solarperformanceinsight_api/routers/parameters.py

```diff
@@ -50,3 +55,37 @@ def get_sandia_inverter(
     return models.SandiaInverterParameters(
         **sandia_inverter_params[inverter_name].to_dict()
     )
+
+
+@router.get(
+    "/cecmoduleparameters",
+    response_model=List[str],
+    responses={200: {"description": "Names of available CEC(SAM) modules"}},
+)
+def list_cec_modules() -> List[str]:
+    """List the names of all Sandia inverters we have parameters for"""
+    return cec_module_params.columns.to_list()
+
+
+@router.get(
+    "/cecmoduleparameters/{module_name}",
+    response_model=models.CECModuleParameters,
+    responses={200: {}, 404: {}, 422: {}},
+    summary="Get a set of CECModuleParameters",
+)
+def get_cec_module(
+    module_name: str = Path(
+        ...,
+        description="Name of the module to fetch parameters for",
+        example="A10Green Technology A10J-S72-175",
```

Comment:
```suggestion
        example="Canadian_Solar_Inc__CS5P_220M",
```
Should also make the app test cover lns 89-90

Reply:
Is this just a more common module?

Reply:
It's roughly the one in the pvlib tutorial, and the previous example was not a valid key so returned 400